Task 1/neerajd007/restapi.py

```diff
@@ -0,0 +1,21 @@
+import requests, re, json
+
+print("Enter the description of the job:")
+desc=raw_input()
+print("Enter the location of the job:")
+location=raw_input()
+resp = requests.get("https://jobs.github.com/positions.json?description"+desc+"&loaction="+location)
```

Comment:
The url to fetch the response is not valid. Spelling of `location` is incorrect and no `=` after `description`. Please fix this

Reply:
@shreyanshdwivedi I have fixed the problem, please review.